cartographer_grpc/map_builder_server.h

```diff
@@ -55,28 +56,51 @@ class MapBuilderServer {
    public:
     MapBuilderContext(MapBuilderServer* map_builder_server);
     cartographer::mapping::MapBuilderInterface& map_builder();
-    cartographer::common::BlockingQueue<std::unique_ptr<SensorData>>&
+    cartographer::common::BlockingQueue<std::unique_ptr<Data>>&
     sensor_data_queue();
     cartographer::mapping::TrajectoryBuilderInterface::LocalSlamResultCallback
     GetLocalSlamResultCallbackForSubscriptions();
-    void AddSensorDataToTrajectory(const SensorData& sensor_data);
+    void AddSensorDataToTrajectory(const Data& sensor_data);
     SubscriptionId SubscribeLocalSlamResults(
         int trajectory_id, LocalSlamSubscriptionCallback callback);
     void UnsubscribeLocalSlamResults(const SubscriptionId& subscription_id);
     void NotifyFinishTrajectory(int trajectory_id);
+    std::unique_ptr<cartographer::mapping::LocalSlamResultData>
+    BuildLocalSlamResultData(
```

Comment:
naming: because it changes unfinished_submaps_ as a side effect, perhaps Read/ReceiveLocalSlamResultData

Reply:
done